stdlib/websub/src/main/java/org/ballerinalang/net/websub/compiler/WebSubServiceCompilerPlugin.java

```diff
@@ -30,11 +43,54 @@
 )
 public class WebSubServiceCompilerPlugin extends AbstractCompilerPlugin {
 
-//    private DiagnosticLog dlog = null;
+    private DiagnosticLog dlog = null;
 
     @Override
     public void init(DiagnosticLog diagnosticLog) {
-//        dlog = diagnosticLog;
+        dlog = diagnosticLog;
+    }
+
+    @SuppressWarnings("unchecked")
+    @Override
+    public void process(ServiceNode serviceNode, List<AnnotationAttachmentNode> annotations) {
+        final UserDefinedTypeNode serviceType = serviceNode.getServiceTypeStruct();
+
+        int webSubAnnotationConfigCount = 0;
+        for (AnnotationAttachmentNode annotation : annotations) {
+            if (ANN_NAME_WEBSUB_SUBSCRIBER_SERVICE_CONFIG.equals(annotation.getAnnotationName().getValue())) {
+                webSubAnnotationConfigCount++;
+                // TODO: 8/19/18 intro annotation validation if required
```

Comment:
Can we add an issue for this so that this won't get missed?

Reply:
Added a comment at https://github.com/ballerina-platform/ballerina-lang/issues/8461#issuecomment-414394978